.mergify.yml

```diff
@@ -0,0 +1,72 @@
+pull_request_rules:
+  - name: Automatic merge for main branch
+    conditions:
+      - base=main
+      - "check-success=Shellcheck"
+      - "check-success=Lint"
+      - "check-success=Tidy"
+      - "check-success=Mocks"
+      - "check-success=Chart checks"
+      - "check-success=Go build"
+      - "check-success=Go test"
+      - "check-success=Test various Envoy + SMI configuration scenarios"
+      - "check-success=Integration Test with Tresor, SMI traffic policies, and egress disabled"
+      - "check-success=Go test e2e (1)"
+      - "check-success=Go test e2e (2)"
+      - "check-success=Go test e2e (3)"
+      - "check-success=Go test e2e (4)"
+      - "check-success=DCO"
+      - "check-success=netlify/osm-docs/deploy-preview"
+      - "#approved-reviews-by>=2"
+      - label=automerge
+      - -label=wip
+      - -label=do-not-merge/hold
+    actions:
+      merge:
+        method: merge
+        strict: true
+
+  - name: Automatic merge for release branch
+    conditions:
+      - base~=^release-
```

Comment:
Is there a reason the main and release branch have separate configs besides regexes? Could one regex capture both like `^(main|release-.*)$`?

Reply:
@nojnhuh it's because [Mergify doesn't support the `or` operator](https://docs.mergify.io/conditions.html#implementing-or-conditions)

Reply:
Could that `or` still be encoded in the regex though?

Reply:
Unfortunately that doesn't work either